src/VirtualClient/VirtualClient.Actions.UnitTests/Examples/StressAppTest/StressAppTestLog_forcedErrors.txt

```diff
@@ -0,0 +1,62 @@
+2022/12/28-08:08:19(UTC) Log: Commandline - ./stressapptest --force_errors -s 30 -l logfile_v8_forcedErrors.txt
```

Comment:
Please make sure to redact some information in case you haven't already done that

Reply:
redacted relevant personal/data related information